### ISSUE TITLE
with talking-heads attention model

### ISSUE THREAD

**Issue by LQchen1** [tensorflow/models#10887]
After adding talking-heads attention to my code, I specifically said a conv2d convolution before and after softmax (attention). The input and output of the convolution are the numbers of heads, and the size of the convolution kernel is 1. But I found that the loss is NAN when the model is running. What is the cause of this?

**Comment by laxmareddyp** [comment]
Hi @LQchen1,

It looks like you haven't used a template to create this issue. Please resubmit your issue using a template from [here](https://github.com/tensorflow/models/issues/new/choose). We ask users to use the template because it reduces overall time to resolve a new issue by avoiding extra communication to get to the root of the issue. We will close this issue in lieu of the new one you will create from the template. Thank you for your cooperation.
